elytron/WFLY-14735-Management-FORM-authentication.adoc

```diff
@@ -0,0 +1,245 @@
+= WFLY-14735 FORM authentication for the management console with MFA
+:author:            Darran Lofthouse
+:email:             darran.lofthouse@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The development of this RFE is delivering a subset of functionality from a much larger planned change,
+this RFE will start with two features:
+ * Enabling FORM authentication for admin console access to the application server.
+ * Where custom security realms are in use supporting MFA authentication on the login form to 
```

Comment:
What is the plan/flow/configuration for this part?

Reply:
I have added a mention of this in the testing section as to how we will illustrate, do you need the sequence adding to the analysis?

Reply:
I've spotted the last paragraph of the _elytron-authentication_ section explains this to some extent. I would like to understand whether the form will be dynamically generated just according to the set of attributes defined in the subsystem, or whether we plan for user to be able to set a custom workflow. In case of the former one explaining sentence would be enough.

Reply:
Ok will add a line, in this case it will be dynamically generated to display the list of configured attributes all in one go.

Later we would add more custom workflow ask for X if Y type things.